### PR TITLE
Camera dev

### PR DESCRIPTION
**Pull request by JeevakaPrabu**
@kalyankondapally @bbeare1

### CODE REVIEW

**Comment by JeevakaPrabu** [comment]
@bjbeare

File: cros_gralloc/gralloc0/cros_gralloc_module.cc [hunk 1]
```diff
@@ -97,7 +97,8 @@ static int cros_gralloc_alloc(alloc_device_t *dev, int w, int h, int format, int
 		supported = mod->driver->is_supported(&descriptor);
 	}
 
-	if (!supported) {
+	if (!supported && !(usage & GRALLOC_USAGE_HW_CAMERA_READ)
+	    && !(usage & GRALLOC_USAGE_HW_CAMERA_WRITE)) {
```
Comment (kalyankondapally) [inline cros_gralloc/gralloc0/cros_gralloc_module.cc]: instead of this we should make this a valid driver combination in i915.c ?

Comment (bjbeare) [inline cros_gralloc/gralloc0/cros_gralloc_module.cc]: Are you OK if we merge this.. as is... in 0.1_things branch and have the commit comment include a REVERTME to fix this properly?

Comment (JeevakaPrabu) [inline cros_gralloc/gralloc0/cros_gralloc_module.cc]: Yes, plan is to include this as a valid driver combination. As the flags were different based on the client and also there is a new combination introduced in V1.0 which will require changes to framework and camera HAL, I would discuss, agree and then do the changes. Thats the main reason, patch has been tagged as HACK

Comment (kalyankondapally) [inline cros_gralloc/gralloc0/cros_gralloc_module.cc]: I am going to prefix the commit to REVERTME: add merge it to master for now.

**Pull request closed without merging**